pywavefront/material.py

```diff
@@ -32,6 +32,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # ----------------------------------------------------------------------------
 
+from sys import stderr
```

Comment:
How about importing logger here? Gives the module importer more flexibility in handling the error.